drip/admin.py

```diff
@@ -148,3 +165,4 @@ class SentDripAdmin(admin.ModelAdmin):
 
 
 admin.site.register(SentDrip, SentDripAdmin)
+
```

Comment:
In order to follow the flake8 style guide of the project, please remove this blank line at end of the file